classes/fullstack/project1/server/src/services/mongodb/build-service.ts

```diff
@@ -0,0 +1,121 @@
+import { BuildCountWeekly, IBuild } from 'types'
+import { BuildModel } from './models'
+import { BUILD_AGREGATION_START_OF_WEEK } from '../../config'
+import { formatISO } from 'date-fns'
+
+export const buildService = {
+  getBuildsPaginated: async ({
+    page = 1,
+    limit = Number.MAX_SAFE_INTEGER,
+  }: Pagination = {}): Promise<IBuildWithId[]> => {
+    try {
+      const raw = await BuildModel.find()
+        .skip((page - 1) * limit)
+        .limit(limit)
+
+      return raw.map((build) => {
```

Comment:
there is no need for this mapping this is the responsibility of the client to correctly display data

Reply:
Corrected to return raw data without any processing, adjusted jest tests